snuba/migrations/snuba_migrations/sessions/0002_sessions_aggregates.py

```diff
@@ -0,0 +1,125 @@
+from typing import Sequence, Tuple
+
+from snuba.clickhouse.columns import (
+    AggregateFunction,
+    Column,
+    String,
+    UInt,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+new_raw_columns: Sequence[Tuple[Column[Modifiers], str]] = [
+    (Column("quantity", UInt(32)), "distinct_id"),
```

Comment:
As discussed offline and as per my comment on the PR that shows we can leave the old aggregated columns empty, I think this should have 1 as default.